src/data/showcases.js

```diff
@@ -1104,6 +1104,15 @@ export const Showcases = [
     source: "https://github.com/cardano-foundation/cardano-governance-voting-tool",
     tags: ["governance", "opensource"],
   },
+  {
+    title: "Finest",
+    description:
+      "Finest envisions a world where the barriers to investing in high-value assets are removed, allowing anyone, anywhere, to participate in markets traditionally reserved for the wealthy and institutional investors.",
```

Comment:
```suggestion
      "Invest in tokenized real world assets. Fully regulated and compliant in Europe.",
```
Suggesting we replace this with two less vague, less emotive phrases from the web site home page.